insuree/tests/test_graphql.py

```diff
@@ -435,3 +435,29 @@ def test_validate_number_validditiy_with_variables(self):
 
         # This validates the status code and if you get errors
         self.assertResponseNoErrors(response)
+        self.assertFalse(content['data']['insureeNumberValidity']['isValid'])
+        
+
+    def test_validate_number_validitiy_with_variables(self):
+        with self.settings(
+                INSUREE_NUMBER_VALIDATOR=None,
+                INSUREE_NUMBER_LENGTH=9,
+                INSUREE_NUMBER_MODULE_ROOT=None):
+            response = self.query(
+              '''
+              query ($insuranceNumber: String!) {
+                insureeNumberValidity(insureeNumber: $insuranceNumber) {
+                  isValid
+                  errorCode
+                  errorMessage
+                }
+              }
+              ''',
+              headers={"HTTP_AUTHORIZATION": f"Bearer {self.admin_token}"},
+              variables={"insuranceNumber": "070707070"})
```

Comment:
Same insurance number as in invalid test. Is invalid test correct? (can't add comment on that)

Reply:
not the second time as I have the with length=9